js/i18n.js

```diff
@@ -73,6 +73,26 @@ function searchForResources(search_url) {
       var time = getSingleItem(params, 'time');
       var manifest = getSingleItem(params, 'manifest');
       var languages = getArrayItem(params, 'lc');
+      
+      var searchFieldArr = [];
+      if(full_text) searchFieldArr.push(full_text);
+      if(repo_name) searchFieldArr.push('repo:'+repo_name);
+      if(user_name) searchFieldArr.push('user:'+user_name);
```

Comment:
We should only support q (full search for now).  These others will not work if they press search.